homeassistant/components/elmax/manifest.json

```diff
@@ -6,7 +6,7 @@
   "documentation": "https://www.home-assistant.io/integrations/elmax",
   "iot_class": "cloud_polling",
   "loggers": ["elmax_api"],
-  "requirements": ["elmax-api==0.0.5"],
+  "requirements": ["elmax-api==0.0.6"],
```

Comment:
Move to 0.0.6.1 which relies to websockets 13.0+: that one should be compatible with most of the other libraries here.